src/cr/cube/cube_slice.py

```diff
@@ -479,6 +480,59 @@ def zscore(self, weighted=True, prune=False, hs_dims=None):
 
         return zscore
 
+    def compare_to_column(self, column=0, weighted=True):
+        """Test statistic and p-value for t-tests of proportions compared
+        to a column
+
+        Returns TtestResult, a pair of arrays of the same size as the current
+        slice: *statistic*, the t-statistic that proportions are different from
+        column *column*; and pvalue, the two-tailed p-value associated with that
+        test statistic (df = table N-2).
+        """
+
+        counts = self.as_array(weighted=weighted)
+        props = self.proportions(axis=0)
+        diff = props[:, [column]] - props
+        margin = self.margin(axis=0, weighted=weighted)
+        var_props = props * (1.0 - props) / margin
+        se_diff = np.sqrt(var_props + var_props[:, [column]])
+        statistic = diff / se_diff
```

Comment:
This line concludes what's presented in Doug's PDF. After this line, the broadcasting and everything, feels a little crammed up. I'd like to see the explicit broadcasting gone, but I'm only reading this for the first time, so I can't really see _why_ it's necessary.

Reply:
and fixes two bugs in his code :P (the direction of the difference, and a very sneaky broadcasting error in R in `/ margin`)